osx_main.py

```diff
@@ -3,8 +3,12 @@
 
 
 def main(start, end, decimal_places):
+    # Make menu item to show what time is set when a user clicks on the percentage
+    menu_item = rumps.MenuItem('Set from {}:00 to {}:00'.format(start, end))
```

Comment:
This will display kinda weird if you put if it's run with `start=7.5, end=5.5` or something like that